docs/setup.md

```diff
@@ -13,9 +13,9 @@ The **Outbreak Response Framework** solution pack depends on the following solut
 
 | Name                     | Type          | Version           | Purpose                                |
 |:-------------------------|:--------------|:------------------|:---------------------------------------|
-| SOAR Framework           | Solution Pack | v2.2.0 and later  | Required for Incident Response modules |
-| Threat Intel Management  | Solution Pack | v1.1.0  and later | Required to ingest threat feeds        |
-| Vulnerability Management | Solution Pack | v1.2.1  and later | Required to ingest CVEs for KEVs     |
+| SOAR Framework           | Solution Pack | v3.0.0 and later  | Required for Incident Response modules |
+| Threat Intel Management  | Solution Pack | v1.2.2  and later | Required to ingest threat feeds        |
+| Vulnerability Management | Solution Pack | v2.1.0  and later | Required to ingest CVEs for KEVs       |
 
 # Configuration
```

Comment:
We will mention about this SP is tested with FortiCloud FAZ 7.4.3 and on-prem 7.4.2

Reply:
not putting this info